ci-operator/templates/openshift/installer/cluster-launch-installer-metal-e2e.yaml

```diff
@@ -408,6 +408,32 @@ objects:
             echo "${CLUSTER_NETWORK_MANIFEST}" > /tmp/artifacts/installer/manifests/cluster-network-03-config.yml
         fi
 
+        if [[ "${CLUSTER_VARIANT}" =~ "numa" ]]; then
```

Comment:
I would start by adding this to the AWS job under multisocket first.  The metal template is out of date and we are in the process of moving it to a new Config, so it’s probably not best place to start.

The other code for multisocket started with this yaml in an earlier state, but note that cluster is a 3 node masters only for cost.

Reply:
ack, thanks.